CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ nav_order: 1
 - Add `aiven_pg_user` import example to docs
 - Extend converter for the service user configuration options `ip_filter` object format
 - Fix `aiven_azure_privatelink_connection_approval` import
+- Add static schema generator for user config options
```

Comment:
Please cover all changes in the changelog, including schema changes (at least most noticeable with a ref to this PR).

Reply:
there weren't any schema changes in particular, we only added a bunch of new fields—I will update the changelog to cover these